### ISSUE TITLE
Invert colours

### ISSUE THREAD

**Issue by joakim000** [cameron/squirt#121]
All the white darn near makes me blind. An option to invert the colour scheme would be very appreciated.

**Comment by dsernst** [comment]
:+1: I'm with you-- would prefer a dark color scheme. :-)

**Comment by Neodelf** [comment]
Yes, invert, and better - add the ability to choose a color scheme. Everyone reads at different days The burden - it would add convenience.

**Comment by mikethepig** [comment]
windows' built-in magnifier?

**Comment by keithwbacon** [comment]
Invert / dark mode would be great
